tutorials/README.md

```diff
@@ -5,8 +5,8 @@ In this folder we collect useful tutorials in order to understand the principles
 
 | Name  | Description   | Type of Problem   |
 |-------|---------------|-------------------|
-| Tutorial1&#160;[[.ipynb](),&#160;[.py](),&#160;[.html]()]| Coming soon |  |
-| Tutorial2&#160;[[.ipynb](tutorial2/tutorial.ipynb),&#160;[.py](tutorial2/tutorial.py),&#160;[.html](http://mathlab.github.io/PINA/_rst/tutorial1/tutorial-1.html)]| Poisson problem on regular domain using extra features | `SpatialProblem` |
-| Tutorial3&#160;[[.ipynb](tutorial3/tutorial.ipynb),&#160;[.py](tutorial3/tutorial.py),&#160;[.html]()]| Wave problem on regular domain using custom pytorch networks. | `SpatialProblem`, `TimeDependentProblem` |
+| Tutorial1&#160;[[.ipynb](tutorial1/tutorial.ipynb),&#160;[.py](tutorial1/tutorial.py),&#160;[.html](http://mathlab.github.io/PINA/_rst/tutorial1/tutorial-1.html)]| Introduction to PINA features |  |
```

Comment:
also the type of problem

Reply:
yes, should I put all of them? We cover them in details for introduction, but in practice (code) we solve a spatial/time problem as example

Reply:
Only the one you use in problem definition! It's just a matter of uniformity!